### PR TITLE
Update service client examples for version 0.1.0 and 1.0.0

### PR DESCRIPTION
**Pull request by kanilsz**

### CODE REVIEW

File: src/MSA.BuildingBlocks.ServiceClient/README.md [hunk 1]
```diff
@@ -11,29 +11,30 @@ To install the package, use the following command in the Package Manager Console
     PM> Install-Package MSA.BuildingBlocks.ServiceClient
 
 ## Usage
-
-First, add the `IServiceClient` to the dependency injection container in `Startup.cs`:
+<details open> <summary>For package version 0.1.0</summary>
+    
+First, add the `ServiceClient` to the dependency injection container in `Startup.cs`:
```
Comment (Marusyk) [inline src/MSA.BuildingBlocks.ServiceClient/README.md:16]: add the `ServiceClient` to the dependency but 
image
you replaced AddServiceClient to AddHttpClient. is that correct?

File: src/MSA.BuildingBlocks.ServiceClient/README.md [hunk 2]
```diff
@@ -11,29 +11,30 @@ To install the package, use the following command in the Package Manager Console
     PM> Install-Package MSA.BuildingBlocks.ServiceClient
 
 ## Usage
-
-First, add the `IServiceClient` to the dependency injection container in `Startup.cs`:
+<details open> <summary>For package version 0.1.0</summary>
```
Comment (Marusyk) [inline src/MSA.BuildingBlocks.ServiceClient/README.md:14]: We don't need to keep info for every version - use git history

Comment (kanilsz) [reply src/MSA.BuildingBlocks.ServiceClient/README.md:14]: True. Going to change.

File: src/MSA.BuildingBlocks.ServiceClient/README.md (hunk 1 above)
Comment (kanilsz) [reply src/MSA.BuildingBlocks.ServiceClient/README.md:16]: For 0.1.0 version is correct. I am going to change it.

**Review by Marusyk — APPROVED** [review]
👍 LGTM. Thank you for your contribution

**Pull request merged**